tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -1587,6 +1585,10 @@ class MklQuantizedConv2DOp
       // Re-scale bias if either of following 2 conditions are met:
       // 1. Bias is not const;
       // 2. Bias is const, but bias cache is empty (first iteration).
+      // TODO Disabling bias caching for now. The graph obtained after
```

Comment:
Nit: Please add one blank line to separate this from the normal comment above.
Nit: TODO should be an action item. Maybe rephrase it?
```c++
// TODO(intel-tf): Re-enable bias caching. Currently, the graph obtained 
// after quantize_graph.py does not run with correct accuracy with this 
// feature enabled.
```